pipeline/pipeline_tasks/api_calls/geocoder.py

```diff
@@ -1 +1,56 @@
-"""Hit Google Maps API to geocode data set"""
+"""
+Geocoding Utility:
+"""
+import os
+
+import pandas as pd
+import sqlalchemy as sqla
+from sqlalchemy import create_engine
+
+from geopy import GoogleV3
+from utilities.db_manager import DBManager
+
+SBA_DWH = os.getenv('SBA_DWH')
+engine = create_engine(SBA_DWH)
```

Comment:
Just for my own edification -- are `SBA_DWH` and `engine` needed here or are they unused?